Microsoft.Toolkit.Parsers/Markdown/Blocks/ListBlock.cs

```diff
@@ -169,8 +169,8 @@ internal static ListBlock Parse(string markdown, int start, int maxEnd, int quot
                         }
                         else
                         {
-                            // Inline text.
-                            AppendTextToListItem(currentListItem, markdown, lineInfo.FirstNonWhitespaceChar, lineInfo.EndOfLine, true);
```

Comment:
@paulbartrum This is a bug I have found where a ListBlockItem will swallow any additional formatting Whitespace, such as used in CodeBlocks.